system/README.md

```diff
@@ -0,0 +1,182 @@
+# System Monitor for Autoware
+
+**Further improvement of system monitor functionality for Autoware.**
+
+## Description
+
+This package provides the following nodes for monitoring system:
+
+- CPU Monitor
+- HDD Monitor
+- Memory Monitor
+- Network Monitor
+- NTP Monitor
+- Process Monitor
+- GPU Monitor
+
+### Supported architecture
+
+- x86_64
+- arm64v8/aarch64
+
+### Operation confirmed platform
+
+- PC system intel core i7
+- NVIDIA Jetson AGX Xavier
+- Raspberry Pi4 Model B
+
+## How to use
+
+Use colcon build and launch in the same way as other packages.
+
+```sh
+colcon build
+source install/setup.bash
+ros2 launch system_monitor system_monitor.launch.xml
+```
+
+CPU and GPU monitoring method differs depending on platform.<br>
+CMake automatically chooses source to be built according to build environment.<br>
+If you build this package on intel platform, CPU monitor and GPU monitor which run on intel platform are built.
+
+## ROS topics published by system monitor
+
+Every topic is published in 1 minute interval.
+
+- [CPU Monitor](docs/topics_cpu_monitor.md)
+- [HDD Monitor](docs/topics_hdd_monitor.md)
+- [Mem Monitor](docs/topics_mem_monitor.md)
+- [Net Monitor](docs/topics_net_monitor.md)
+- [NTP Monitor](docs/topics_ntp_monitor.md)
+- [Process Monitor](docs/topics_process_monitor.md)
+- [GPU Monitor](docs/topics_gpu_monitor.md)
+
```

Comment:
Please add
```
 - [Voltage Monitor)(doc/topics_voltage_monitor.md)
```
and describe `/diagnostic` topic which the voltage_monitor publishes in `topics_voltage_monitor.md`.

Reply:
Added description.